cpp/lib/src/MarcUtil.cc

```diff
@@ -166,9 +167,13 @@ bool Record::recordSeemsCorrect(std::string * const err_msg) const {
     }
 
     if (raw_record_.length() > 99999) {
+	/*
         *err_msg = "record length (" + std::to_string(raw_record_.length())
                    + ") exceeds maxium legal record length (99999)!";
+	*/
         return false;
+	Warning("record length (" + std::to_string(raw_record_.length())                                                         
```

Comment:
First returning and then trying to print a warning will not work as expected ;-)